appintro/src/main/java/com/github/appintro/indicator/ProgressIndicatorController.kt

```diff
@@ -1,12 +1,12 @@
-package com.github.appintro.indicator
+package dev.appintro.core.indicator
```

Comment:
This is using the wrong package declaration. The file is in `com.github.appintro`